DEVELOP.md

```diff
@@ -2,12 +2,27 @@
 
 ## Documentation
 
-OUDS Android documentation is generated using [Writerside](https://www.jetbrains.com/writerside/).
-In order to modify the OUDS Android documentation or test it locally, you need to [download and install it](https://www.jetbrains.com/writerside/download/).
+OUDS Android documentation is generated using [Dokka](https://github.com/Kotlin/dokka).
 
-When installed, launch Writerside and open the `ouds-android/docs` directory.
+You can test the documentation locally by launching the `dokkaGenerate` Gradle task.
 
-In the Writerside table of content, click on "Getting started". The Writerside Preview panel is on the right with an "Open in browser" button to test
-documentation display in a browser.
+```
+./gradlew dokkaGenerate
+```
+
+When generated, open `ouds-android/docs/dokka/index.html` in your browser.
```

Comment:
Opening the `index.html` directly does not fully work because the lef side column is not visible. We should use the link provided by Dokka at the end of the task: http://localhost:63342/OUDS%20Android/docs/dokka/index.html. But I don't know if the port is always the same.